apps/np-atobarai/src/modules/atobarai/atobarai-money.ts

```diff
@@ -0,0 +1,9 @@
+export class AtobaraiMoney {
+  public readonly amount: number;
+  public readonly currency: string;
+
+  constructor(amount: number, currency: string) {
+    this.amount = amount;
+    this.currency = currency;
+  }
+}
```

Comment:
1. you need private constructor to enforce nominal typing
2. maybe we can try to use zod object + brand() as well?